featurebyte/service/feature_manager.py

```diff
@@ -309,12 +313,16 @@ async def remove_online_store_compute_queries(self, feature_spec: OnlineFeatureS
                     # OnlineStoreComputeQueryService
                     pass
 
-    async def remove_online_store_cleanup_jobs(self, feature_spec: OnlineFeatureSpec) -> None:
+    async def remove_online_store_cleanup_jobs(
+        self, session: Optional[BaseSession], feature_spec: OnlineFeatureSpec
```

Comment:
Curious why the session type is `Optional`?

Reply:
This is to handle the case when no session can be established, in which case it will be passed as None.